configure.ac

```diff
@@ -1,12 +1,33 @@
 AC_PREREQ([2.61])
 AC_INIT([mstpd], [0.0.4], [https://github.com/mstpd/mstpd])
 
+# bridge-stp must be installed in /sbin/, and the associated scripts and
+# executables should also be installed outside of /usr/.  Set the default
+# exec_prefix so they are installed in the correct places by default.
+# AC_PREFIX_DEFAULT(...) can be used to override the default prefix, but there
+# doesn't seem to be any standard way to override the default exec_prefix, so
+# just do it manually.
+test "x$exec_prefix" = xNONE && exec_prefix=''
```

Comment:
Yes, `./configure --exec-prefix=/sbin/` works.  However, prior to this commit, running `./configure` without any arguments will cause bridge-stp to be installed as /usr/local/sbin/bridge-stp ... Which means the usual `./autogen.sh ; ./configure ; make ; sudo make install` will not do what the user expects.  With this commit, `./configure` without any arguments will cause bridge-stp to be installed as /sbin/bridge-stp ... although the user can still override this using `./configure --exec-prefix=...` if they want.